pkg/util/docker/dockerfile/builder/client.go

```diff
@@ -280,9 +280,12 @@ func (e *ClientExecutor) LoadImage(from string) (*docker.Image, error) {
 		return nil, docker.ErrNoSuchImage
 	}
 
-	repository, _ := docker.ParseRepositoryTag(from)
+	repository, tag := docker.ParseRepositoryTag(from)
+	if len(tag) == 0 {
+		tag = "latest"
```

Comment:
This package can't take any dependencies on other OpenShift code, because we're going to move this out of the repo.